### DISCUSSION TITLE
PriveVue 4: impossible to set title attribute and aria-label to the Dialog close and maximize buttons

### DISCUSSION THREAD

**Discussion by xavier-ottolini** [primefaces/community#2877] [PrimeVue] [answered]
Currently, it is only possible to set the aria-label and the title to the "close" and "maximize" buttons with the `closeButtonProps` and the `maximizeButtonProps`. But it changes the buttons classes. So the buttons look and feel change.
It would be better to have properties that does not change those buttons behavior.

**Comment by cagataycivici** [comment, marked as answer]
Great use case for Pass-Through attributes;

```html
<Dialog pt:pcCloseButton:root:ariaLabel="'Edit Profile'">
```

**Reply by xavier-ottolini** [reply]
Hi,
It works ! The ` pt` property is not very documented. I did not know how to use it. 
Thank you very much!
